source/reference/minio-mc/mc-batch-generate.rst

```diff
@@ -256,52 +249,48 @@ Sample YAML
      apiVersion: v1
```

Comment:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude

Might be helpful for single-sourcing the sample YAML?

Reply:
👍